src/accelerate/commands/launch.py

```diff
@@ -472,6 +499,10 @@ def launch_command(args):
     # Use the proper launcher
     if args.use_deepspeed and not args.cpu:
         deepspeed_launcher(args)
+    elif args.use_fsdp and not args.cpu:
+        if version.parse(torch.__version__) < version.parse("1.12.0.dev20220418+cu113"):
+            raise ValueError("FSDP requires PyTorch >= 1.12.0.dev20220418+cu113")
```

Comment:
Was it not out in 1.11 ?. Let's use the `base_version` attribute to compare to 1.12.0 without anything added ;-)

Reply:
As mentioned offline, at present the model saving/checkpointing is supported through nightly version only.